.github/workflows/check.yml

```diff
@@ -54,7 +54,17 @@ jobs:
 
       - name: kairos x86_64-linux
         if: matrix.os == 'ubuntu-latest'
-        run: nix build -L --no-link --show-trace .#packages.x86_64-linux.kairos
+        run: nix build -L --out-link kairos-result --show-trace .#packages.x86_64-linux.kairos
+
+      - name: Create JUnit Kairos Test Report x86_64-linux
+        if: matrix.os == 'ubuntu-latest'
+        run: cat kairos-result/test-result.json | nix run nixpkgs#cargo2junit > kairos-test-result.xml
```

Comment:
Using `cargo2junit` forces us to use nigthly toolchain :disappointed:.

What about using `cargo nextest` instead of default cargo test runner? Nextest has JUnit output and it was designed for usage in CI.